### PR TITLE
EVS-1071: Added power consumption to transaction ended messages

### PR DESCRIPTION
**Pull request by MounirKaroudEV**

### CODE REVIEW

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java [hunk 1]
```diff
@@ -156,9 +146,7 @@ NotifyReportRequest createNotifyReportRequest(@Nullable Integer requestId, boole
 
     private TransactionEventRequest createTransactionEvent(Integer evseId, Integer connectorId, TransactionEventRequest.TriggerReason reason, TransactionData transactionData,
                                                            TransactionEventRequest.EventType eventType, Instant currentDateTime, Long seqNo) {
-        List<MeterValue> meterValues = Collections.singletonList(new MeterValue().withTimestamp(ZonedDateTime.now(ZoneOffset.UTC)).withSampledValue(
-                Collections.singletonList(new SampledValue().withValue(eventType == STARTED ? ZERO : new BigDecimal("1010")))));
-        return createTransactionEvent(evseId, connectorId, reason, transactionData, eventType, currentDateTime, seqNo, meterValues);
+        return createTransactionEvent(evseId, connectorId, reason, transactionData, eventType, currentDateTime, seqNo, 1000);
```
Comment (akoshterek) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java]: hardcoded value? I think we have a proper counter for consumption

Comment (MounirKaroudEV) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java]: Yes but in some cases there is no values. It's better if i don't send anything then

Comment (akoshterek) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java]: wait, but you do still send 1000 instead of nothing

Comment (MounirKaroudEV) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java]: Yes, i will change the pr :)

**Comment by MounirKaroudEV** [comment]
Yes i will, and this github ui is not the best to review

**Pull request merged**